framework/src/media/BufferInputDataSource.cpp

```diff
@@ -81,13 +77,14 @@ bool BufferInputDataSource::open()
 bool BufferInputDataSource::close()
 {
 	unregisterDecoder();
+	unregisterStream();
```

Comment:
I think, stop() firstly, and then unregisterStream().

Reply:
Thanks for your review. I fixed it.